src/ethereum/__init__.py

```diff
@@ -14,11 +13,3 @@
 #
 EVM_RECURSION_LIMIT = 1024 * 12
 sys.setrecursionlimit(max(EVM_RECURSION_LIMIT, sys.getrecursionlimit()))
-
-
-def evm_trace(evm: Any, op: Any) -> None:
-    """
-    Placeholder for an evm trace function. The spec does not trace evm by
-    default. EVM tracing will be injected if the user requests it.
-    """
-    pass
```

Comment:
Hm... interesting idea. The current `evm_trace` implementation is certainly simple enough that it isn't a problem to include.

The reason we had the placeholder and put the implementation in `ethereum_spec_tools` was to clearly differentiate between what is specification and what is not. Moving the implementation into `ethereum/` would imply that it is somehow spec related.

Why do you want to move it?

Reply:
@SamWilsn updated as discussed in the sync i.e. marking the doctoring with a tag.